lib/String.fz

```diff
@@ -39,6 +39,10 @@ public String ref : property.equatable, property.hashable, property.orderable is
   # returns true if string is empty or contains whitespace only
   public is_blank => utf8 ∀ u -> u.is_ascii_white_space
 
+  # true if this string represents an object String that is null
+  # when passed on to the Java interface
+  module is_java_null_string => false
```

Comment:
I would prefer if this where not in String but in fuzion.java:
```
fuzion.java is
...
    module String.is_java_null_string => false
    public String.type.java_null_string String => ...
```
Not sure if this is possible?...